packages/project/src/json/v4-specification.json

```diff
@@ -921,11 +928,7 @@
         "Field": {
             "additionalProperties": false,
             "description": "A field defining annual crop management practices. Fields are defined by geographic boundaries that contain crop management practices that are identical across the whole of that boundary.",
-            "errorMessage": {
-                "_": "projectDataError:fieldUnknownError",
-                "additionalProperties": "projectDataError:fieldUnknownAdditionalProperty",
-                "required": "projectDataError:fieldRequiredPropertyMissing"
-            },
+            "errorMessage": "```js\n{\n\"required\": \"projectDataError:fieldRequiredPropertyMissing\",\n\"additionalProperties\": \"projectDataError:fieldUnknownAdditionalProperty\",\n\"_\": \"projectDataError:fieldUnknownError\"\n}\n```",
```

Comment:
this looks unexpected

Reply:
removed the js comment wrapper around errorMessage annotations.